src/com/keep/calm/enjoy/java/a/primitive/datattpe/parnia/assignment/While/Loop.java

```diff
@@ -0,0 +1,14 @@
+package com.keep.calm.enjoy.java.a.primitive.datattpe.parnia.assignment.While;
+
+import com.keep.calm.enjoy.java.a.primitive.datattpe.operator.precedence.Main;
+
+public class Loop {
+
```

Comment:
You should put your code in a main method :
public static void main(String[] args) {
        int i = 0;
    while (i < 5) {
        System.out.println(i);
        i++;
        }
    }